### ISSUE TITLE
Testing fails

### ISSUE THREAD

**Issue by ennitje** [silverstripe-scienceninjas/embargoexpiry#2]
Testing with `phpunit embargoexpiry/tests/`fails.
The first error
```shell
PHP Fatal error:  Object::add_extension() - Can't find extension class for "EmbargoExpiryExtension" in /var/www/www.kempinski.com/htdocs/framework/core/Object.php on line 537
```
can be fixed by updating the `$requiredExtensions` property like
```php
protected $requiredExtensions = array(
		'SiteTree' => array(
			'WorkflowEmbargoExpiryExtension',
			'Versioned',
		)
	);
```
At the second error i'm not so sure. 
```shell
PHP Fatal error:  Class 'WorkflowAction' not found in /var/www/www.kempinski.com/htdocs/embargoexpiry/code/actions/PublishItemWorkflowAction.php on line 10
```

The class `PublishItemWorkflowAction`extends from `WorkflowAction` but `WorkflowAction` doesn't exists. So maybe this class is obsolete and can be deleted? Same for the test `testPublishActionWithFutureDates` and `createDefinition`? Or did i miss something.

**Comment by mparkhill** [comment]
Hello

Yes the workflow action class is redundant, was left over from the work
reducing the advanced workflow module.

Feel free to submit a pull request for both those issues, I'm happy to
review and merge them.

Thanks,
Michael

On Thursday, April 16, 2015, ennitje <notifications@github.com> wrote:

> Testing with phpunit embargoexpiry/tests/fails.
> The first error
>
> PHP Fatal error:  Object::add_extension() - Can't find extension class for "EmbargoExpiryExtension" in /var/www/www.kempinski.com/htdocs/framework/core/Object.php on line 537
>
> can be fixed by updating the $requiredExtensions property like
>
> protected $requiredExtensions = array(        'SiteTree' => array(            'WorkflowEmbargoExpiryExtension',            'Versioned',        )    );
>
> On the second error i'm not so sure.
>
> PHP Fatal error:  Class 'WorkflowAction' not found in /var/www/www.kempinski.com/htdocs/embargoexpiry/code/actions/PublishItemWorkflowAction.php on line 10
>
> The class PublishItemWorkflowActionextends from WorkflowAction but
> WorkflowAction doesn't exists. So maybe this class is absolete and can be
> deleted? Same for the test testPublishActionWithFutureDates and
> createDefinition? Or did i miss something.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/silverstripe-scienceninjas/embargoexpiry/issues/2>.
>


-- 
Michael Parkhill
Developer / SilverStripe
021551965

**Comment by ennitje** [comment]
Hi Michael,
thanks for your answer. I submitted a pull request.
Regards,
Julius

**Comment by mparkhill** [comment]
Issue #1 also fixed with 931246e, thanks.
